python/ray/tune/integration/wandb.py

```diff
@@ -308,6 +320,7 @@ def __init__(
         api_key: Optional[str] = None,
         excludes: Optional[List[str]] = None,
         log_config: bool = False,
+        save_checkpoints: bool = True,
```

Comment:
why is it default to true?

Reply:
Let's chat with the W&B team on what they think is a sensible default and what their users expect. I believe most people who use an experiment tracking service would appreciate the models being logged there as well

Reply:
Synced with the team and decided to go with `False` per default. Thanks for the suggestion!

Reply:
Thanks for raising with the team. I saw the response. Yeah let's go with False for now!